Bio/GenBank/Scanner.py

```diff
@@ -476,16 +477,16 @@ def parse_records(self, handle, do_features=True):
         """
         # This is a generator function
         while True:
-            record = self.parse(handle, do_features)
-            if record is None:
+            next_record = self.parse(handle, do_features)
```

Comment:
Given the loop only deals with a single record, why use ``next_record`` rather than ``record`` as the variable name?